### PR TITLE
Dockerfile and docker-compose setup for mongo and node

### PR DESCRIPTION
**Pull request by wfortin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,8 +8,16 @@ const cors = require('cors')
 const passport = require('passport')
 
 const mongoose = require('mongoose')
-const mongoUri = process.env.MONGOLAB_URI || 'mongodb://localhost/ubeat'
-mongoose.connect(mongoUri)
+const mongoUri =
+  process.env.MONGOLAB_URI || `mongodb://${process.env.DATABASE_URL}` || 'mongodb://localhost/ubeat'
+console.log(mongoUri)
```
Comment (vseguin) [inline index.js]: Don't log an URI with potential creds i'd say

File: index.js [hunk 2]
```diff
@@ -8,8 +8,16 @@ const cors = require('cors')
...
+mongoose.connect(
+  mongoUri,
+  {
+    autoReconnect: true,
+    reconnectTries: Number.MAX_VALUE
```
Comment (vseguin) [inline index.js]: Hmmm why so? Not sure to see the benefit there

File: index.js (hunk 1 above)
Comment (wfortin) [reply index.js]: Oops yes. It was for testing purposes, will remove this.

File: index.js (hunk 2 above)
Comment (wfortin) [reply index.js]: It's if the database connection fails, just have it retry, but it should probably be set to like 10. Was also when I was debugging issues with docker. I'll remove it and test again.

**Pull request merged**